hrm-lib/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "hrm-lib",
```

Comment:
I'd prefer to have modules named for their purpose rather than using generic terms like 'lib' or 'common' or 'shared' - when modules have those sort of names people see it as a free pass to throw anything & everything in there! 

If this is going to be purely for shared types, something like `hrm-types` or `hrm-contracts` or something like that

Reply:
I prefer local modules for stuff that's only consumed within one repo, because:

* Working within a single repo has far less friction for developers, I know there's ways to mitigate the friction but it's never as good as the code all being in the same place
* There's no versioning headaches

Once you move code out to a separate repo it goes from being a shared module, to being a library, which I think brings a whole load more responsibilities in terms of documentation, CI / CD infrastructure, change management etc.

Since local modules are cheap, we could feasibly have 2, one for types, another for SSM caching and more for whatever other shared code we want. That would mean any application that only wanted the types wouldn't need to pull in a bunch of SSM client dependencies etc.

Reply:
That makes total sense to me!